components/tag/Tag.razor.cs

```diff
@@ -16,22 +18,32 @@ public partial class Tag : AntDomComponentBase
         [Parameter]
         public string Mode { get; set; } = "default";
 
+        //Here we keep the orginal string so we can support custom colors and inverse
         [Parameter]
-        public string Color
+        public OneOf<string, TagColor> Color
```

Comment:
We can add another parameter `PresetColor` for the enum

Reply:
Should we do a check to see if both Preset Color and Color have been set and throw an exception?

Reply:
I think exceptions should be thrown as little as possible. A better practice is to roll back to Color, and make a note of it in the documentation.